packages/collaboration-manager/src/CollaborationManager.ts

```diff
@@ -112,7 +119,7 @@ export class CollaborationManager {
    * Redo last undone operation in the local stack
    */
   public redo(): void {
-    this.#currentBatch?.terminate();
```

Comment:
It's not clear why in both cases (undo and redo) there's a call `moveBatchToUndo`

Reply:
Maybe `putBatchToUndo` would be better